src/endpoints/getCollectionCount.ts

```diff
@@ -6,6 +6,7 @@ export const getCollectionCount: PayloadHandler = async (req, res) => {
     if (!req.user) return res.sendStatus(401);
     const collectionName = req?.body?.collectionName;
     const payloadQuery = req?.body?.query;
+    payloadQuery["tenant"] = req.headers.host;
```

Comment:
This count operation does not obey access rules on `read` (which only applies to `find` and `findById` therefore server side we add this filter